src/main/java/com/superzanti/serversync/config/SyncConfig.java

```diff
@@ -25,7 +25,7 @@
     public List<String> FILE_IGNORE_LIST = Arrays.asList("**/serversync-*.jar", "**/serversync-*.cfg");
     public List<String> CONFIG_INCLUDE_LIST = new ArrayList<>();
     public Locale LOCALE = Locale.getDefault();
-    public ETheme THEME = ETheme.BLUE_YELLOW;
```

Comment:
Looks like an unrelated change to logging. Should probably not change this anyway as it is the UI author's original intent and user-configurable.